profileViewer/modelViewer.js

```diff
@@ -194,6 +206,8 @@ const ModelViewer = {
   initialize: () => {
     canvasParentElement = document.getElementById('modelViewer');
     assetErrorElement = document.getElementById('assetError');
+    assetErrorList = document.createElement('ul');
```

Comment:
Why not not just make the assetError element be a UL instead of a div so you don't need to manage it here?

Reply:
Ah, yes, well you see, that was for the very good reason that I, in fact, did.... not.... think of that. 😅